src/en/developer-guide/applications/utilities/custom-assignment-cascade.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TALXIS Custom Assignment Cascade
+tagline: Enables advanced assignment cascade configurations
+author: František Čapek
+---
+
+# **Assignment Cascade Configuration**
+
+This extension enables custom settings for one-to-many relationship assignment cascade. Use this when you need to make advanced filtering or if you want to use different attribute other than the default _ownerid_. TALXIS Custom Assignment Cascade is part of the Ownership feature.
+
+## Important!
+**You cannot create a configuration for a relationship that has enabled the default assignment cascade in the metadata, change the Assign Cascade to Cascade-None first if you want to use it.**
+
+# Entity
+
+## **TALXIS Configuration - Assignment Cascade**
+| Attribute name 	| Description 	| Type | Requirement
+|-	|-	|- |-
+| talxis_configuration_assignmentcascadeid 	| Unique id.	| Guid | System
+| talxis_filteringattributesavedstate 	| State of filtering attribute - pre-save or post-save.	| Two options | Optional
+| talxis_filterownershipattributename 	| Filtering attribute of type systemuser/team. 	| String | Optional
+| talxis_name 	| Name of the configuration. | String | Required
+| talxis_pluginstepid 	| Id of the associated sdk message processing step. | String (Guid) | Read-only
+| talxis_recordset 	| Fetch xml filter query. The logic will apply only to these records. Default query is generated after the record is saved.	| String | Optional
+| talxis_relationshipname 	| Logical name of entity relationship. This one-to-many relationship must exist between the source and the target entity. 	| String | Required
+| talxis_sourceentityname 	| Logical name of the source (triggering) entity.| String | Read-only
+| talxis_sourceownershipattributename 	| Logical name of the source (triggering) attribute of type systemuser/team. Leave empty to use default value = 'ownerid'.	| String | Required
+| talxis_targetentityname 	| Logical name of the target entity.	| String | Read-only
+| talxis_targetownershipattributename 	| Logical name of the target attribute of type systemuser/team. Leave empty to use default value = 'ownerid'. 	| String | Required
+| StateCode 	| Entity state. Values are Active/Inactive. 	| State | System
```

Comment:
I might also write that inactive records means the logic does not trigger

Reply:
Good point.